packages/addons-website/scss/components/_overlay.scss

```diff
@@ -0,0 +1,65 @@
+@import '../prefix';
+
+.#{$prefix}--overlay-content {
+  bottom: 0;
+  left: 0;
+  right: 0;
+  top: 0;
+  padding: 2rem;
+  position: fixed;
+  display: flex;
+  flex-direction: row;
+  z-index: 17000;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.#{$prefix}--overlay-leftArea {
+  align-self: flex-start;
+  flex-basis: 20%;
+  z-index: 17001;
+}
+
+.#{$prefix}--overlay-centerArea {
+  align-self: center;
+  flex-basis: 60%;
+  z-index: 17002;
+}
+
+.#{$prefix}--overlay-rightArea {
+  align-self: flex-start;
+  color: $ibm-colors__white;
+  flex-basis: 20%;
+  z-index: 17003;
+  text-align: right;
+  
+  svg {
+    flex-basis: auto;
+  }
+}
+
+.#{$prefix}--overlay-image {
+  height: auto;
+  display:block;
+  margin:auto;
+  max-height: 100%;
+  max-width: 100%;
+  object-fit: contain;
+  user-select: none;
+}
```

Comment:
Can we add `background: $ui-background` here, then should be good to go! 🎉

Reply:
updated.